index.html

```diff
@@ -5,8 +5,41 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
 </head>
 <body>
+    <header class="titleBlock">
+        <h1 id="siteTitle">The Weather Station</h1>
+    </header>
+    <span class="banner" id="skyBanner"></span>
+    <section class="cityNameBlock">
```

Comment:
Consider refactoring your class names so that they are lowercase and hyphenated between words
```html
<h1 id="site-title">The Weather Station</h1>
```